readme.md

```diff
@@ -80,6 +90,17 @@ public interface Enumerable<X> extends Collection<X> {
         // ...
     }
 
+    /**
+     * Passes at least one element of the collection to the each given blocks.
+     * If no predicate (null) is given, then true is returned instead.
+     * @param first The predicate to match at least one element.
+     * @param other The array of predicates to match at least one element.
+     * @return The true if the blocks never return false or nil.
+     */
+    default boolean any(Predicate<X> first, Predicate<X>... other) {
```

Comment:
please remove the obsolete methods from `readme.md` also

Reply:
done